packages/dashboards/src/lib/configurator/services/configurator-heading.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+
+@Injectable()
```

Comment:
could we add a `providedIn: "root"` here? That way we don't need to register the provider in the individual modules. I think we do this for most of the dashboard providers already.

Reply:
okay, makes sense 👍